revscoring/dependencies/__init__.py

```diff
@@ -1,3 +1,22 @@
+"""
+This module provides a general set of utilities for implementing set of
+dependencies, solving then and performing injections.
+
+.. automodule:: revscoring.dependencies.dependent
```

Comment:
This is correct, but it would be better if None were the default value.  I'll make the change in the next update.